### ISSUE TITLE
Get status of currently (re)publishing IPNS names

### ISSUE THREAD

**Issue by lockedshadow** [ipfs/go-ipfs#4815]
#### Version information:
go-ipfs version: 0.4.13-
Repo version: 6
System version: 386/linux
Golang version: go1.9.2

#### Type:
Enhancement

#### Area:
Commands

#### Description:
I use IPNS for publish updates of some... mutable resource. But sometimes, when my IPFS daemon are terminated unexpectedly (e.g. due to OOM killer, which happens pretty often for me), it seems like no longer republishing previously published IPNS names after re-launch. Is not very easy to quickly figure out why my IPNS name is not resolving *again*: is it a random network fluctuations, or my publisher node got in trouble... So it will be useful to see a status of currently published names. 
Even in case that IPFS daemon will be working without unexpected terminating, it will be great to get some kind of information about currently published names. Something like `ipfs name status` command. It should be can at least...

- Select of which IPNS name to get status about. (`ipfs name status --key=mykey` to get status of `mykey` by analogy of `ipfs name publish` command.)
- Show whether is publishing something (or not) to a certain key. If yes, then show which IPFS path it is. (`Currently publishing: None` or `Currently publishing: /ipfs/QmSomething`)
- Show date and time when the previous publishing was performed, and when the next one is scheduled.
- Show parameters with which the previous publishing was performed. (Lifetime and TTL.)
- If opted, try resolve the selected IPNS name and show whether the resolved value is the same as published. (`--resolve=true`)
- And show errors, if their occurred during the previous publishing.

**Comment by Stebalien** [comment]
I agree. We definitely should have a command for listing the status of what keys we're currently publishing. However, we shouldn't *forget* to that we're publishing something (and we always republish 5m after restarting).

You can call `ipfs name resolve  -n ...` to resolve without looking at your local cache. That'll tell you if other nodes can still access your IPNS record.

**Comment by lockedshadow** [comment]
> You can call `ipfs name resolve -n ...` to resolve without looking at your local cache. That'll tell you if other nodes can still access your IPNS record.

It useful for debugging purpose, thanks!

**Comment by kuzdogan** [comment]
I came up with the following workaround. It basically prints all keys (i.e. names) and resolves each of them.
```
$ ipfs key list -l | cut -d ' ' -f1 | xargs -L1 ipfs name resolve -n
```
